src/olympia/urls.py

```diff
@@ -43,6 +44,8 @@
     re_path(r'^uploads/', include(upload_patterns)),
     # Downloads.
     re_path(r'^downloads/', include(download_patterns)),
+    # Activity.
+    re_path(r'^activity/', include(attachment_patterns)),
```

Comment:
I'd lean towards putting the url under `developers/` but don't know if @diox  has a counter opinion.

Reply:
`activity/` makes slightly more sense to me, it's an activity attachment after all, and it's available to reviewers, not just developers.

The cleaner way would then be to:
- Move the existing `activity/urls.py` to `activity/api_urls.py`
- Re-create the `activity/urls.py` file and add a fresh `urlpatterns` definition to it with that new URL
- Include it with `re_path(r'activity/', include('olympia.activity.urls'))` here.

Note that you would also need to add `activity` to `SUPPORTED_NONAPPS` and `SUPPORTED_NONLOCALES` settings (as well as the equivalent addons-frontend config variables)

Reply:
(I was mainly thinking of a way of avoiding having to add another top level app/folder to configurations everywhere)

Reply:
It's a little bit annoying but I think that's worth doing to avoid for consistency (and the less reviewers functionality depend on developer URLs/views/etc the better IMHO)

Reply:
> Note that you would also need to add activity to SUPPORTED_NONAPPS and SUPPORTED_NONLOCALES settings (as well as the equivalent addons-frontend config variables)

I did notice that doing this caused the download link to no longer work, but I'm assuming its due to the changes not being synced?